.github/workflows/hardhat-network-tracing-scheduled.yml

```diff
@@ -0,0 +1,44 @@
+name: Hardhat Network Tracing Capabilities, scheduled runs for all solc verisons
+
+on:
+  schedule:
+    - cron: "0 0 * * *"
+  workflow_dispatch:
+
+defaults:
+  run:
+    working-directory: packages/hardhat-core
+
+concurrency:
+  group: ${{github.workflow}}-${{github.ref}}
+  cancel-in-progress: true
+
+jobs:
+  test_tracing:
+    name: Test tracing capabilities on Ubuntu with Node ${{ matrix.node }}
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node: [14, 16, 18]
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: ${{ matrix.node }}
+          cache: yarn
+      - name: Install
+        run: yarn --frozen-lockfile
+      - name: Build
+        run: yarn build
+      - name: Cache artifacts
+        uses: actions/cache@v2
+        with:
+          path: |
+            packages/hardhat-core/test/internal/hardhat-network/stack-traces/test-files/**/artifacts
+          key: hardhat-network-stack-traces-tests-${{ hashFiles('yarn.lock') }}-${{ hashFiles('packages/hardhat-core/test/internal/hardhat-network/stack-traces/test-files/**/*.sol') }}-${{ hashFiles('packages/hardhat-core/test/internal/hardhat-network/stack-traces/test-files/**/test.json') }}-${{ hashFiles('packages/hardhat-core/test/internal/hardhat-network/stack-traces/**/*.ts') }}
+      - name: Run tests
+        env:
+          DO_NOT_SET_THIS_ENV_VAR____IS_HARDHAT_CI: true
+          FORCE_COLOR: 3
+          NODE_OPTIONS: "--max-old-space-size=4096"
+        run: yarn test:tracing
```

Comment:
First I started reading ["Reusing workflows"](https://docs.github.com/en/actions/using-workflows/reusing-workflows), but I quickly got annoyed with how much content there was to wade through, and I realized that I could just hack this into place by basically duplicating the existing workflow file and making a few small tweaks, so that's what i did :sweat_smile: